src/Api/Client.php

```diff
@@ -68,6 +68,8 @@ protected function requestToArray($request)
         $requestArray['connect_timeout'] = $request->getConfiguration(Plugin::API_CONFIGURATION_KEY_CONNECTION_TIME_OUT) ?? self::DEFAULT_CONNECTION_TIME_OUT;
         $requestArray['timeout'] = $request->getConfiguration(Plugin::API_CONFIGURATION_KEY_TIME_OUT) ?? self::DEFAULT_CONNECTION_TIME_OUT;
 
+        $this->logger->error('requestToArray', $requestArray);
```

Comment:
Please go through the code and remove any log lines that were needed for debugging.